src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1747,7 +1747,7 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                         do iCell = 1, nCellsAll
 
                            ! Reset tracer1 to 2 in top n layers
-                           do k = 1, config_reset_debugTracers_top_nLayers
+                           do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers
```

Comment:
For this loop, if `minLevelCell(iCell)` is set to 1, then you are looping from `1` to `1+config_reset_debugTracers_top_nLayers`.  Previously, the loop was from `1` to `config_reset_debugTracers_top_nLayers`.  Is the desired behavior to change the number of iterations of this loop for the case when `minLevelCell(iCell) == 1`?

Reply:
Good catch. No, we do want the same number of iterations:
```suggestion
                           do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers-1
```